### ISSUE TITLE
unable to highlight if there is a bold tag in text

### ISSUE THREAD

**Issue by emilianos** [vivekneupane11/reactjs-editor-demo#1]
if you apply bold to a couple of words in the text and then try to highlight the whole paragraph it doesnt work
I guess it wont work on more complex html with anchor tags and so on

**Comment by vivekneupane11** [comment]
Yes. When it comes to applying bold to a couple of words in the text and
then try to highlight the whole paragraph, it has to be the other way
around. Meaning you can  highlight the whole paragraph first and then make
text bold.

This library does not yet support complex structures like <a> tag yet.
